runtime/core/pom.xml

```diff
@@ -72,4 +72,35 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    
+    <build>
```

Comment:
Why is this required? atlas-mode is already in dependency

Reply:
Hi Tomo,
A jar file is needed to test AtlasUtil.findClassesFromJar method. copied atlas-model jar into target folder for Unit testing. Thanks

Reply:
Ah OK... not really ideal to make POM fat with test only configuration though, I'd say OK for now.